functions/callPythonFunction/pythonFunctions/create_certificates.py

```
@@ -265,6 +292,103 @@ def fetch_template_text(self, enrollment):
+                    except (ValueError, AttributeError):
+                        ects_formatted = "0"
+                    
+                    formatted_entry = f"{course_title} ({program_title}) ({ects_formatted} ECTS)"
+                    formatted_participations.append(formatted_entry)
+            
+            # Events are added at the end
+            formatted_participations.extend(events)
+            
+            return formatted_participations
+            
+        except Exception as e:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)